common/test/acceptance/tests/lms/test_lms_edxnotes.py

```diff
@@ -255,6 +268,63 @@ def test_can_delete_notes(self):
         components = self.note_unit_page.components
         self.assert_notes_are_removed(components)
 
+    def test_can_create_note_with_tags(self):
+        """
+        Scenario: a user of notes can define one with tags
+        Given I have a course with 3 annotatable components
+        And I open the unit with 2 annotatable components
+        When I add a note with tags for the first component
+        And I refresh the page
+        Then I see that note was correctly stored with its tags
+        """
+        self.note_unit_page.visit()
+
+        components = self.note_unit_page.components
+        for note in components[0].create_note(".{}".format(self.selector)):
+            note.tags = ["fruit", "tasty"]
+
+        self.note_unit_page.refresh()
+        self.assertEqual(["fruit", "tasty"], self.note_unit_page.notes[0].tags)
+
+    def test_can_change_tags(self):
+        """
+        Scenario: a user of notes can edit tags on notes
+        Given I have a course with 3 components with notes
+        When I open the unit with 2 annotatable components
```

Comment:
Nit: the repetitive "Given I have a course with 3 components..." seems unnecessary. Very few of these tests depend on having a course with 3 components, and if we add a fourth it will be annoying to have to update every string.